test_runner/fixtures/metrics.py

```diff
@@ -61,9 +61,8 @@ def parse_metrics(text: str, name: str = "") -> Metrics:
     "pageserver_smgr_query_seconds_bucket",
     "pageserver_smgr_query_seconds_count",
     "pageserver_smgr_query_seconds_sum",
-    "pageserver_storage_operations_seconds_bucket",
-    "pageserver_storage_operations_seconds_count",
-    "pageserver_storage_operations_seconds_sum",
+    "pageserver_storage_operations_seconds_count_total",
+    "pageserver_storage_operations_seconds_sum_total",
```

Comment:
Need `pageserver_storage_operations_seconds_global_{bucket,count,sum}` here as well.

Reply:
Thanks, good point, need to test those too. They aren't per tenant anymore, but I'll craft a separate one for those.